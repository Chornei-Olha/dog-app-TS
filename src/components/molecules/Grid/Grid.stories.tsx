import type { Meta } from '@storybook/react';

import { Grid } from '.';

const meta = {
  title: 'Molecules/Grid',
  component: Grid
} satisfies Meta<typeof Grid>;

export default meta;

export const MainGrid = {};
