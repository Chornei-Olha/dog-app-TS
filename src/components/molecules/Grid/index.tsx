import { Box, styled } from '@mui/material';

import { GridCard } from '../GridCard';
import { useGetImagesQuery } from '../../../services/images';

const List = styled(Box)(() => ({
  display: 'grid',
  gridTemplateColumns: '1fr 1fr 1fr 1fr',
  gridColumnGap: '20px',
  gridRowGap: '20px',
  gridAutoFlow: 'dense'
}));

const Item = styled(Box)(() => ({
  width: '100%',
  height: '100%',
  minWidth: '200px',
  gridRow: 'span 1',

  '&:nth-of-type(1)': {
    gridColumn: '1/3;',
    gridRow: '1/3'
  },

  '&:nth-of-type(2)': {
    gridColumn: '3 / 4',
    gridRow: 'span 1'
  },

  '&:nth-of-type(3)': {
    gridColumn: '4 / 5',
    gridRow: '1/3'
  },

  '&:nth-of-type(4)': {
    gridColumn: '3 / 4',
    gridRow: 'span 2'
  },

  '&:nth-of-type(5)': {
    gridColumn: '1 / 2',
    gridRow: 'span 2'
  },

  '&:nth-of-type(10)': {
    gridColumn: '4 / 5',
    gridRow: 'span 1'
  }
}));

export const Grid = () => {
  const { data: images } = useGetImagesQuery({ limit: 10, page: 0 });
  return (
    <List>
      {(images || []).map(item => (
        <Item key={item.id}>
          <GridCard imgUrl={item.url} />
        </Item>
      ))}
    </List>
  );
};
