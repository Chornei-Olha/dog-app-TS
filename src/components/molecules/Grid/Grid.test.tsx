import { composeStories } from '@storybook/react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import * as stories from './Grid.stories';

describe('MainGrid', () => {
  const { List } = composeStories(stories);
  it('should match snapshot MainGrid', () => {
    const { container } = render(
      <BrowserRouter>
        <List />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
