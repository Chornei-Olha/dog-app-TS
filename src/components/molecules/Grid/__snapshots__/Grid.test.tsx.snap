// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should match snapshot MainGrid 1`] = `
.emotion-0 {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  grid-column-gap: 20px;
  grid-row-gap: 20px;
  grid-auto-flow: dense;
}

.emotion-1 {
  width: 100%;
  height: 100%;
  min-width: 200px;
}

.emotion-1:nth-of-type(1) {
  grid-column: 1/3;
  grid-row: span 2;
}

.emotion-1:nth-of-type(2) {
  grid-column: 3/4;
  grid-row: span 1;
}

.emotion-1:nth-of-type(3) {
  grid-column: 4/5;
  grid-row: span 2;
}

.emotion-1:nth-of-type(4) {
  grid-column: 3/4;
  grid-row: span 2;
}

.emotion-1:nth-of-type(5) {
  grid-column: 1/2;
  grid-row: span 2;
}

.emotion-1:nth-of-type(10) {
  grid-column: 4/5;
  grid-row: span 1;
}

.emotion-2 {
  box-shadow: rgba(146, 31, 237, 1);
  height: 100%;
  border-radius: 20px;
  height: 100%;
  box-shadow: 6px 6px 0px 0px rgba(0, 0, 0, 1);
  -webkit-transition: box-shadow 0.3s ease-in-out;
  transition: box-shadow 0.3s ease-in-out;
}

.emotion-2:hover {
  box-shadow: 6px 6px 8px 0px rgba(25, 118, 210, 1);
}

.emotion-2:active {
  box-shadow: 6px 6px 8px 0px rgba(25, 118, 210, 1);
}

.emotion-3 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);
  overflow: hidden;
  border-radius: 20px;
  box-shadow: none;
  position: relative;
  height: 100%;
}

.emotion-4 {
  display: block;
  -webkit-background-size: cover;
  background-size: cover;
  background-repeat: no-repeat;
  -webkit-background-position: center;
  background-position: center;
  width: 100%;
  object-fit: cover;
  height: 100%;
  object-fit: cover;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  width: 100%;
  height: 100%;
  opacity: 0;
  -webkit-transition: opacity 0.3s ease-in-out;
  transition: opacity 0.3s ease-in-out;
}

.emotion-5>:not(:first-of-type) {
  margin-left: 8px;
}

.emotion-5:hover {
  opacity: 1;
  background-color: rgba(0, 0, 0, 0.2);
}

.emotion-6 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  text-align: center;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  font-size: 1.5rem;
  padding: 8px;
  border-radius: 50%;
  overflow: visible;
  color: rgba(0, 0, 0, 0.54);
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -ms-flex-pack: end;
  -webkit-justify-content: flex-end;
  justify-content: flex-end;
  width: 100%;
}

.emotion-6::-moz-focus-inner {
  border-style: none;
}

.emotion-6.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-6 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-6:hover {
  background-color: rgba(0, 0, 0, 0.04);
}

@media (hover: none) {
  .emotion-6:hover {
    background-color: transparent;
  }
}

.emotion-6.Mui-disabled {
  background-color: transparent;
  color: rgba(0, 0, 0, 0.26);
}

.emotion-7 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-9 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.875rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 16px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #fff;
  background-color: #9c27b0;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
}

.emotion-9::-moz-focus-inner {
  border-style: none;
}

.emotion-9.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-9 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-9:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #7b1fa2;
  box-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);
}

@media (hover: none) {
  .emotion-9:hover {
    background-color: #9c27b0;
  }
}

.emotion-9:active {
  box-shadow: 0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);
}

.emotion-9.Mui-focusVisible {
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
}

.emotion-9.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  box-shadow: none;
  background-color: rgba(0, 0, 0, 0.12);
}

<div>
  <div
    class="MuiBox-root emotion-0"
  >
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/238/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/239/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/242/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root emotion-1"
    >
      <div
        class="MuiBox-root emotion-2"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root emotion-3"
        >
          <img
            alt="item"
            class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img emotion-4"
            src="https://picsum.photos/id/237/570/419"
          />
          <div
            class="MuiCardActions-root MuiCardActions-spacing emotion-5"
          >
            <button
              aria-label="add to favorites"
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium emotion-6"
              tabindex="0"
              type="button"
            >
              <test-file-stub
                aria-hidden="true"
                classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                focusable="false"
              />
              <span
                class="MuiTouchRipple-root emotion-7"
              />
            </button>
            <div
              class="MuiBox-root emotion-8"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium emotion-9"
                tabindex="0"
                type="button"
              >
                Goldendoodle
                <span
                  class="MuiTouchRipple-root emotion-7"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
